src/views/capture-view.ts

```diff
@@ -139,10 +137,10 @@ export default class CaptureView extends View {
   }
 
   private async storeResult(blob: Blob) {
-    const buffer = await blobToArrayBuffer(blob);
-    const record = new ImageRecord(buffer);
-    const id = await db.store(record);
-    router.visit(`/edit/${id}`);
+    const record = new ImageRecord();
+    record.setOriginal(blob);
+    await record.save();
```

Comment:
Where do you catch `save()` rejection errors?

Reply:
A quick look shows that this is an area that I need to do a sweep over generally. I've filed a separate issue (#32) to look into it in a new PR, partly because I think part of that work is going to involve using the pubsub system in #31 to display some errors to the user.